server/handlers/people/edit-view.js

```
@@ -13,7 +13,7 @@ module.exports = function (request, reply, source, joiErr) {
   var getActiveOrgs = request.server.methods.pg.organisations.getActive;
   var options, msg;
 
-  if (loggedIn.userId !== editId && loggedIn.scope !== 'admin') {
+  if (JSON.parse(loggedIn.userId) !== editId && loggedIn.scope !== 'admin') {
     msg = 'You do not have the permissions to edit this user\'s settings';
 
     return reply(Boom.badRequest(msg));
```

We need to update the type of the boom error. It is not a "badRequest" (400 html status code) but a permission restriction so we can return a forbidden error and we don't need to specify anymore the message error as it is done on the  [hapi-error-config file](https://github.com/emfoundation/ce100-app/blob/master/server/hapi-error-config.js

So I think it should be
```
return reply(Boom.forbidden());
```
@mk4111 do you mind to update this line on this PR also?